cypress/e2e/registration.cy.js

```diff
@@ -1,11 +1,95 @@
 /// <reference types='cypress' />
+let user;
 
 describe('Student Registration page', () => {
   before(() => {
-
+    cy.visit('/automation-practice-form');
+    cy.task('generateUser').then((u) => { user = u; });
   });
 
   it('', () => {
+    cy.get('#firstName')
+      .type(user.firstName);
```

Comment:
It would be nice to add a custom command for finding elements by placeholder and use it where it is needed

Reply:
oh, I didn't change it, I had this command before, but I'm using finding by placeholder just once per homework, other times I'm using finding by id, so I thought it would be just extra rows of code. But if I'm not right tell me, I'll change it

Reply:
At least for practice that would be nice to add